packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -109,7 +109,7 @@ export const addOrUpdateIssueComment = async (
   for (const comment of listCommentsResponse.data) {
     if (comment.body?.includes(commentMark)) {
       // We found the existing comment, so updating it
-      await github.issues.updateComment({
+      await github.rest.issues.updateComment({
```

Comment:
In the  newest version of `probot`, `github` is moved under the `rest` namespace.